ui/client/datasets/annotations/MultiColumnGeoSelector.js

```diff
@@ -23,12 +23,16 @@ const MultiColumnGeoSelector = ({
   // we hardcode the column's value (adminX option) to the columName being edited
   // and even disable that field (see disabled prop on the react tree below).
   useEffect(() => {
+    // TODO: I can't figure out how to hit even the console.log outside of the if statement
```

Comment:
@ccjoel I can't figure out how to hit this useEffect - maybe you have an idea?

Reply:
This is not for multi column using two coordinates (lat lon)

but instead for a multipart annotation that consists of for columns, one of each:
country, admin1, admin2, and admin3

If you have a dataset with 4 geo columns that match that, and you start an annotation in one where you select one of them to be of any of those types (not coordinates nor latitude or longitude) you can toggle on the checkbox to start this process and render this component.

Reply:
![image](https://github.com/jataware/dojo/assets/1967061/d4bc86f9-c056-4efd-a886-54f0cbed6f63)